recovery/recovery.py

```diff
@@ -188,6 +195,10 @@ def main(scriptConf=None):
             redisConnection.reload_modules_cache()
             redisConnection.reload_vendors_cache()
             LOGGER.info('All the modules data set to Redis successfully')
+            job_log_messages.extend([
+                {'label': 'Loaded modules', 'message': len(modules)},
+                {'label': 'Loaded vendors', 'message': len(vendors)}
+            ])
```

Comment:
It is not really important to job_log how many modules was loaded. However, if we want to log this information we need to distinguish between `load` and `save` -> see current dashboard in admin ui, there is `recovery - save`